spec/features/todo_items_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe "Todo items" do
+   before do
+      @user = create(:user)
+      visit(user_path(@user))
+    end
+  
```

Comment:
Any suggestions on how to test creation & deletion of items? Having trouble finding the rendered form_field to test user  adding new task, also would like to test user pressing "Enter" to create items. 
I also am having trouble locating the element of the delete button, to test deleting an item? How would you approach this issue?